openproblems/tasks/multimodal_data_integration/methods/harmonic_alignment.py

```diff
@@ -7,9 +7,19 @@
 import sklearn.decomposition
 
 
-def _harmonic_alignment(adata, n_svd=100, n_eigenvectors=100, n_pca_XY=100):
```

Comment:
any reason to remove the explicit passing of parameters here?

Reply:
So right now we never use this explicit passing of parameters anywhere -- having them set manually by `test` gives the author an option of setting different defaults for testing vs benchmarking.

One option if we want to keep explicit parameter setting as a possibility is

```
def my_method(adata, test=False, my_param=None):
    if test:
        my_param = my_param or 100
    else:
        my_param = my_param or 1000
    # do stuff
```

Reply:
I see. I think if we would want to re-use these methods in other contexts (e.g., via the CLI you set up), then it would be useful to be able to set the parameters as well. I quite like the `my_param` setup.